### ISSUE TITLE
cDNA_annotation_comparer.dbi task interrupt

### ISSUE THREAD

**Issue by sanjin725** [PASApipeline/PASApipeline#323]
I have a question. The cDNA_annotation_comparer.dbi task was terminated due to a power outage. I deleted the generated files and then ran the script again. Will the files generated before the power outage in the SQLite database affect the results of the new task?
xiaoxin pei

**Comment by alpapan** [comment]
Yes, I think it usually does. What I do is make a gz backup of the sqlite database before running comparer. (actually after every Launch_PASA command)

**Comment by alpapan** [comment]
yeap. the second time you run it will check if the updated gene models should be merged with existing cdna alignments. recommendation is to run the cdna comparer twice, once with the first output of pasa, then with the output of the previous run. The second run takes days sadly. putting the sql database on a ramdrive (/tmp or /dev/shm) helps a lot. glad it worked out for you!
